package.json

```diff
@@ -31,8 +31,10 @@
     "link:local": "node bin/yarn-link-local"
   },
   "resolutions": {
-    "typescript": "3.3.3",
-    "broccoli-typescript-transpiler/typescript": "3.3.3"
+    "typescript": "3.5.3",
+    "amd-name-resolver": "https://github.com/ember-cli/amd-name-resolver.git",
```

Comment:
I think this requires the changes from https://github.com/ember-cli/amd-name-resolver/pull/27, right?